src/openrct2/scripting/bindings/world/ScTileElement.cpp

```diff
@@ -607,16 +628,19 @@ namespace OpenRCT2::Scripting
             duk_push_null(ctx);
         return DukValue::take_from_stack(ctx);
     }
-    void ScTileElement::mazeEntry_set(uint16_t value)
+    void ScTileElement::mazeEntry_set(const DukValue& value)
     {
-        ThrowIfGameStateNotMutable();
-        auto el = _element->AsTrack();
-        if (el != nullptr)
-            if (get_ride(el->GetRideIndex())->type == RIDE_TYPE_MAZE)
-            {
-                el->SetMazeEntry(value);
-                Invalidate();
-            }
+        if (value.type() == DukValue::Type::NUMBER)
+        {
+            ThrowIfGameStateNotMutable();
+            auto el = _element->AsTrack();
+            if (el != nullptr)
```

Comment:
We generally always use braces if the nested block has more than one line.

Reply:
As you can see, I just left it how it was.

Reply:
well can you change it.